src/iris/ui/templates/plan.html

```diff
@@ -324,6 +325,7 @@ <h4><i class="icon-plus icon">+</i> Add application tracking template</h4>
     <p class="item-content" title="{{ secondsToMinutes wait }}"><label class="item-label light" data-toggle="tooltip" title="Minimum time that must elapse before repeating or moving onto the next escalation step.">Wait</label> {{ secondsToMinutes wait }} minutes</p>
     <p class="item-content" title="{{ count }}"><label class="item-label light" data-toggle="tooltip" title="Number of times to execute this notification before escalating to next step">Count</label> {{ count }} times</p>
     <p class="item-content" title="{{ template }}"><label class="item-label light" data-toggle="tooltip" title="Content to use for the notification">Template</label> <a href="/templates/{{template}}">{{ template }}</a></p>
+    <p class="item-content" title="{{ optional }}"> <label class="item-label light" data-toggle="tooltip" title="If set to true, this notification won't generate an alert if it fails to find a valid target.">Optional</label> {{#isEqual optional 1}}True{{else}}False{{/isEqual}}</p>
```

Comment:
You can probably do normal `{{#if optional #}}` instead of explicitly comparing it to 1 as 1 is a truthy value.

Reply:
Changed.